lib/tasks/invalid_incidents.rake

```diff
@@ -0,0 +1,12 @@
+namespace :incidents do
+  desc 'fix invalid incidents'
+  task update_incidents: :environment do
+    completed = Incident.where( completed: true )
+    completed.each do |incident|
+      if !incident.valid?
+        incident.completed = false
```

Comment:
We should make sure to coordinate this with the affected staff. That is, they should be given a warning that they're about to have ~100 uncompleted incidents.

Reply:
ooof yeah good point.